[31-Jul-2024 16:08:05 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 16:08:05 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('http://3.14.37....', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 16:08:49 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php?route=/&amp;route=%2F failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 16:08:49 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php?route=/&route=%2F failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/php...', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 16:09:08 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 16:09:08 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/php...', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:55:50 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/localhost/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:55:50 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/localhost/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/loc...', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:56:27 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/ failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:56:27 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/ failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/php...', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:57:29 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:57:51 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:57:53 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:57:55 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:18 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:20 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:23 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:25 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:26 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:58:28 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:59:07 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[31-Jul-2024 17:59:07 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin/index.php failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/php...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 13:39:02 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 13:51:43 UTC] PHP Warning:  mysqli::__construct(): php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 13:51:43 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 3.14.37.218/phpMyAdmin failed: No such host is known.  in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli->__construct('3.14.37.218/php...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 14:00:40 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli_connect('3.14.37.218:330...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 14:15:55 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli_connect('3.14.37.218:330...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[01-Aug-2024 20:16:23 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function executableexec() in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php:7
Stack trace:
#0 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php on line 7
[01-Aug-2024 20:18:08 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function scriptexec() in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php:15
Stack trace:
#0 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php on line 15
[01-Aug-2024 20:35:57 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php:45
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php(45): mysqli_connect('127.0.0.1', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php on line 45
[01-Aug-2024 20:38:49 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php:45
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php(45): mysqli_connect('127.0.0.1', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php on line 45
[02-Aug-2024 13:47:45 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli_connect('3.14.37.218:330...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[02-Aug-2024 13:48:24 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php:4
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php(4): mysqli_connect('3.14.37.218:330...', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\test_connect.php on line 4
[02-Aug-2024 14:46:55 UTC] PHP Fatal error:  Uncaught mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php:45
Stack trace:
#0 C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php(45): mysqli_connect('127.0.0.1', 'root', 'Rkakfsus97313', 'detroit')
#1 {main}
  thrown in C:\inetpub\wwwroot\DQSDirect_test_database_connection\init_ssh_tunnel.php on line 45
